### ISSUE TITLE
Inviting R community and event organizers from Africa and Asia to the RUG slack.

### ISSUE THREAD

**Issue by DragonflyStats** [forwards/tasks#32]
I propose we make contact with R community and event organizers to join the R user group slack.

This is not exactly with the scope of Forwards duties, but it is an easy job, and would yield benefits down in the future.

**Comment by madlene** [comment]
I agree that these communities are important to connect with. I would be happy to work with you on this.

**Comment by hturner** [comment]
I think this is in scope!

**Comment by hturner** [comment]
This would help with encouraging satRdays in places outside US and Europe c.f. #34.

**Comment by hturner** [comment]
There are other resources that groups should be aware of such as the R User Groups Support Program: https://www.r-consortium.org/projects/r-user-group-support-program.

Maybe this task could extend to working on a starter kit similar to https://github.com/rladies/starter-kit/wiki and promoting that to RUGs in countries where the R community is less established.
